tardis/adapters/sites/slurm.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, machine_type, site_name):
                                                                       CANCELLED=ResourceStatus.Stopped,
                                                                       COMPLETED=ResourceStatus.Stopped,
                                                                       DEADLINE=ResourceStatus.Stopped,
-                                                                      FAILED=ResourceStatus.Error,
```

Comment:
I would suggest to keep it in ResourceStatus.Error, since we would like track and not hide errors. Instead errors should be handled correctly by the Adapter.